prog/vnet/load_balancer_nexus.rb

```diff
@@ -60,8 +60,20 @@ def before_run
     if load_balancer.need_certificates?
       nap 60
     else
+      load_balancer.vms.each do |vm|
+        bud Prog::Vnet::UpdateLoadBalancerNode, {"subject_id" => vm.id, "load_balancer_id" => load_balancer.id}, :put_certificate
+      end
+      hop_wait_cert_broadcast
+    end
+  end
+
+  label def wait_cert_broadcast
+    reap
+    if strand.children_dataset.where(prog: "Vnet::UpdateLoadBalancerNode").all? { _1.exitval == "certificate server is started" } || strand.children.empty?
```

Comment:
this seems a little weird because `reap` already deletes stuff. It seems like you'd always be falling into the `empty?` case? are there other children?

Reply:
The health probes are also a child strand of load balancer strand.

Reply:
given `reap` loads `children` (right? I read the source code last night, but it's particular to use `delete_if` to keep it in sync), I'd probably use the in-memory data structure instead.